packages/content/src/Domain/Model/ExcerptTrait.php

```diff
@@ -125,10 +125,17 @@ public function getExcerptCategories(): array
     public function setExcerptCategories(array $excerptCategories): void
     {
         $this->initializeCategories();
-        $this->excerptCategories->clear();
```

Comment:
We should introduce a PHPStan rule for this clear makes in our setUp always problems 🙈 As second flush in our domain event listener (which is on postFlush event) has the unit of work of previous flush not completely cleaned up.